example/src/App.tsx

```diff
@@ -1299,6 +1303,24 @@ class App extends React.Component<{}, State> {
             resizable
             // onChange={this.valueUpdater('appTextCounter')}
           />
+                                <Card>
+                                    <CardBody componentTitle="Online store dashboard - Card" sectioned>
```

Comment:
@HiirenP What's the use of componentTitle here? now since we already have `CardHeader`, we can use that to show the title right?

Reply:
I have seen at many places, where header is used as heading of card in general and componentTitle is used as subTitle of card body and that can be multiple as no. of sections in card body. but should i still remove it according to you if will not be useful in future?

Reply:
@HiirenP yeah I don't see the use of it, lets remove it